### PR TITLE
release/public-v2: final updates to Docker documentation before release

### PR DESCRIPTION
**Pull request by climbfuji**
This PR contains the following changes:

- fix formatting in final version of doc/README_docker_gnu.md

No testing required, documentation updates only. Will need to updatee the tag ufs-v2.0.0 after this PR is merged.

Waiting for final OK from @mkavulich before merging.

### CODE REVIEW

**Review by SamuelTrahanNOAA — APPROVED** [review]
Formatting is correct now.

File: doc/README_docker_gnu.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Short-Range Weather App Public Release in Docker
+
 This document explains how to run the SRW public release, version 1,
 in a Docker container. We will run twice: once by building an image,
 and a second time manually.
```
Comment (mkavulich) [inline doc/README_docker_gnu.md]: >We will run twice: once by building an image, and a second time manually.

I feel that this wording is confusing, especially for users not intimately familiar with both docker *and* the UFS SRW App. I'd suggest changing the wording to something like:

>We will run twice: once by building a Docker image complete with the application, and a second time building the application manually within a pre-existing Docker image.

File: doc/README_docker_gnu.md [hunk 2]
```diff
@@ -151,11 +150,11 @@ The files:
    set the LAYOUT_X, LAYOUT_Y, RUN_CMD_UTILS, and RUN_CMD_POST
    variables.
 
-3. LOW MEMORY MACHINES - The workflow takes about 16 GB of RAM, on top
-   of the memory your OS and other applications use. If you don't have
-   significantly more than 16 GB of RAM, then use the 4 core config,
-   but reduce the utilities to one MPI rank. Do that by putting this
-   at the end of config.sh:
+3. LOW MEMORY MACHINES - The workflow used more than 16 GB of RAM, on
```
Comment (mkavulich) [inline doc/README_docker_gnu.md]: Change to "*uses* more than 16 GB of RAM".

A side note: my machine only has 16 GB of RAM, but I was able to run the app successfully with the RAM+swap space allocated to Docker (20 GB total). Not sure if that's worth a mention though.

File: doc/README_docker_gnu.md [hunk 3]
```diff
@@ -424,8 +421,7 @@ You will see the final job, the post, finish its 48th hour:
     + (( i<=48 ))
 
 
-6. Where is my Output?
-----------------------
+## Section 6: Where is my Output?
```
Comment (mkavulich) [inline doc/README_docker_gnu.md:428]: Did we take out the graphics generation from the latest version of the Docker container? When I ran ./run_all.sh it ended with the graphics generation and an informational message about where to view the output.

File: doc/README_docker_gnu.md [hunk 4]
```diff
@@ -466,8 +462,7 @@ you will see a great many files:
 
 
 
-7. Changing the Code
---------------------
+## Section 7: Changing the Code
 
 To do actual development, you want to compile manually instead of
 using the `ufs-srweather-app-Dockerfile`. There is extensive guidance
```
Comment (mkavulich) [inline doc/README_docker_gnu.md:473]: Can we link to the App and Model users guides here?

**Review by mkavulich — CHANGES_REQUESTED** [review]
Github won't allow me to comment specifically on lines that haven't been modified, so sorry about the dump of comments here:

 - **Line 8** Needs something after "and"... maybe "get the premade container and input data"?
 - **Line 32** Remove comma
 - **Line 172** compress --> decompress
 - **Line 187** I should have mentioned this earlier, but the given example is *not* a linux-friendly path like we want users to create. Can we change to something like `/home/example_home_directory/ufs`?
 - **Line 195** include the word "data" in front of "archives"
 - **Line 512** Is this correct? Should it not be `module load NCEPLIBS/2.0.0`?

**Comment by climbfuji** [comment]
@mkavulich I addressed all your comments except the question about the graphics (chapter 6). I will need @SamuelTrahanNOAA to help me with that one.

File: doc/README_docker_gnu.md (hunk 3 above)
Comment (SamuelTrahanNOAA) [reply doc/README_docker_gnu.md:428]: I'd like to keep section 6 in, so people know where the output is even if they close the terminal that had the run_all.sh output.

**Comment by SamuelTrahanNOAA** [comment]
@climbfuji @mkavulich - I'd like to keep section 6 in, so people know where the output is even if they close the terminal that had the run_all.sh output.

**Comment by mkavulich** [comment]
@SamuelTrahanNOAA I'm not advocating for omitting section 6, I'm curious if the run_all script still includes the graphics generation (as it did when I ran my tests last week). If so, the graphics should be mentioned in section 5 and/or 6, if not, I'm okay with the PR as-is.

**Comment by SamuelTrahanNOAA** [comment]
The graphics are still in the container. I wrote the `README.md` before adding the graphics. I thought I had added all necessary details since then, but I guess I forgot some.

**Comment by SamuelTrahanNOAA** [comment]
@mkavulich - Dom just merged some changes from one of my PRs that updates sections 5 & 6. Please review them again.

**Review by mkavulich — APPROVED** [review]
Looks great, thanks @climbfuji and @SamuelTrahanNOAA for the quick work!

**Pull request merged**